### ISSUE TITLE
Two many false positive

### ISSUE THREAD

**Issue by Gasp34** [AlexeyAB/darknet#4093]
Hello, 

I trained YOLO to detect 9 class of animals, it works very fine and it almost always detect the good animal.. when there is one of the 9 class on the image. 
But when I run the detection on an image with another species, a human or a vehicle it often detect something while it should not. Even if I try with a random picture of a house with a totally different background, it detects something!

There are 8000 images for the training (with only the 9 species on it) and they are taken 20 different locations (so most of the images have the same background). 

Do you know why this happens and how to fix it? 
I understood that I can add negative example, but does I need some negative examples for each object that could be in front of the camera ? they would be to many...

Here are the loss and map graph and one example.
Thanks !

![chart](https://user-images.githubusercontent.com/38688186/66831686-8fc93700-ef58-11e9-9b90-b5146f65082f.png)

![foto_no_exif](https://user-images.githubusercontent.com/38688186/66831848-e8003900-ef58-11e9-9404-7b996944bffb.jpg)

**Comment by AlexeyAB** [comment]
https://github.com/AlexeyAB/darknet#how-to-improve-object-detection

desirable that your training dataset include images with non-labeled objects that you do not want to detect - negative samples without bounded box (empty .txt files) - use as many images of negative samples as there are images with objects
